storageClient/api.go

```diff
@@ -87,6 +113,9 @@ type GroupMember struct {
 // DeleteGroupMembersRequest  wraps the information of all members being removed from group provided.
 type DeleteGroupMembersRequest AddGroupMembersRequest
 
+// GetMembershipKeyRequest wraps the information to get the membership key for a client
+type GetMembershipKeyRequest AddGroupMembersRequest
```

Comment:
Only type alias if the data is literally the same. I don’t think to get a membership key we need to send along a field for GroupMembers